### PR TITLE
Migrate Dashboard table to the new version on table components #2682

### PR DESCRIPTION
**Pull request by David-DB88**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Migrate Dashboard table to the new version on table components.

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/common/NewTable/LinkTopic.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      style={{ color: 'blue' }}
+      to={clusterTopicsPath(row.original.name)}
+      title={value}
+      onClick={handleClick}
+    >
+      {value}
+    </NavLink>
+  );
+};
+
```
Comment (workshur) [inline kafka-ui-react-app/src/components/common/NewTable/LinkTopic.tsx:21]: I don't like the approach when we create toipc link component. Let's try to create universal component where we will send `to` as a property

File: kafka-ui-react-app/src/components/Dashboard/ClustersWidget/__test__/ClustersWidget.spec.tsx [hunk 2]
```diff
@@ -34,10 +34,4 @@ describe('ClustersWidget', () => {
   it('when cluster is read-only', () => {
     expect(screen.getByText('readonly')).toBeInTheDocument();
   });
-
-  it('render clusterWidget cells', () => {
-    const cells = screen.getAllByRole('cells');
-    expect(cells.length).toBe(14);
-    expect(cells[0]).toHaveStyle('max-width: 99px');
-  });
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Dashboard/ClustersWidget/__test__/ClustersWidget.spec.tsx:42]: Not sure why we need to remove this test case

**Review by workshur — CHANGES_REQUESTED** [review]
Looks good so far but still need some changes

File: kafka-ui-react-app/src/components/common/NewTable/LinkTopic.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const LinkTopic = ({ row, to }: any) => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/common/NewTable/LinkTopic.tsx:5]: let's call it LinkCell and update all the existing tables with same link components

**Review by workshur — CHANGES_REQUESTED** [review]

File: kafka-ui-react-app/src/components/Dashboard/ClustersWidget/ClustersWidget.tsx [hunk 4]
```diff
@@ -28,6 +27,30 @@ const ClustersWidget: React.FC = () => {
...
+      { header: 'Version', accessorKey: 'version' },
+      { header: 'Brokers count', accessorKey: 'brokerCount' },
+      { header: 'Partitions', accessorKey: 'onlinePartitionCount' },
+      {
+        header: 'Topics',
+        accessorKey: 'topicCount',
+        // eslint-disable-next-line react/no-unstable-nested-components
+        cell: ({ row }) => (
+          <LinkCell
+            color="blue"
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Dashboard/ClustersWidget/ClustersWidget.tsx:42]:
```suggestion
           
```

**Review by workshur — CHANGES_REQUESTED** [review]

**Comment by Haarolean** [comment]
@armenuikafka please take a look at the new dashboard, everything should look (almost) the same without any functionality issues.

**Review by armenuikafka — CHANGES_REQUESTED** [review]
Please make the "Topics" link design visible for user.
https://user-images.githubusercontent.com/104780608/194303908-1f2450d2-9a81-4999-aec9-b34503d86131.mov

**Comment by Haarolean** [comment]
@armenuikafka @David-DB88 let's just get rid of this link at all

**Comment by armenuikafka** [comment]
> @armenuikafka @David-DB88 let's just get rid of this link at all

@Haarolean do not redirect to Topics?

**Comment by armenuikafka** [comment]
> Custom deployment will be available at http://pr2694.internal.kafka-ui.provectus.io

@David-DB88 , Topics count link still exists.

**Comment by David-DB88** [comment]
but I don't see the link anywhere @armenuikafka 
Screen Shot 2022-10-06 at 19 58 53

**Comment by armenuikafka** [comment]
@David-DB88 I'm checking http://pr2694.internal.kafka-ui.provectus.io/.    

https://user-images.githubusercontent.com/104780608/194479582-507e5128-403b-41c6-b184-2833c33c59a6.mov

**Review by workshur — APPROVED** [review]

**Comment by Haarolean** [comment]
@armenuikafka a redeploy did help it seems, PTAL

**Review by armenuikafka — APPROVED** [review]

**Pull request merged**